LICENSE

```diff
@@ -1,17 +1,15 @@
-#######
-License
-#######
+check_po, python package to automate task on traduction file .po
+Copyright (C) 2016  aRkadeFR
```

Comment:
I'm not very well aware of licensing and process around it. I only use GPLv3 license for my projects, as it is recommended by the fsf.
I'm free to amend it or change it again.